src/pixel.rs

```diff
@@ -28,6 +30,9 @@ impl Rgba {
             alpha,
         })
     }
+    fn as_image_rgba(&self) -> image::Rgba<u8> {
```

Comment:
Btw, I prefer to have an extra newline between definitions. I would like to just have it enforced by `rustfmt` but there doesn't seem to be an existing configuration option.